www/src/components/account/billing/BillingPricingCards.tsx

```diff
@@ -95,23 +98,30 @@ function BillingPricingCards() {
               checked: true,
             },
           ]}
-          callToAction={isProPlan ? (
-            <Button
-              secondary
-              width="100%"
-              onClick={isEnterprisePlan ? null : () => setDowngradeModalOpen(true)}
-            >
-              Downgrade
-            </Button>
-          ) : (isEnterprisePlan ? <ContactUs /> : renderCurrentPlanButton())}
+          callToAction={
+            isProPlan ? (
+              <Button
+                secondary
+                width="100%"
+                onClick={
+                  isEnterprisePlan ? null : () => setDowngradeModalOpen(true)
+                }
+              >
+                Downgrade
+              </Button>
+            ) : isEnterprisePlan ? (
+              <ContactUs />
+            ) : (
+              renderCurrentPlanButton()
```

Comment:
Could we refactor this into another function component?  These `useCallback` subcomponents are crazily bug-prone